### ISSUE TITLE
On activity page show long text with read more along with the image

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtMediaWP/rtMedia#1535]
```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2
rtMedia: v4.5.11
```

**Steps to Reproduce:**
1. Go to activity page
2. Enter long text along with the image
3. Observe that if long text is entered then read more is shown but the image is shown only after clicking on read more. 
We need to show the long text truncated with a read more option and the image displayed below the truncated text.

Refer to the Visual -> https://drive.google.com/file/d/1ARhvuXPYg3LteBn0ERqLCeL7nViPolK8/view

**Comment by dishitpala** [comment]
`[Read More]` is coming from BuddyPress default functionality 

we need to stop truncating the text by using 
```
add_filter( 'bp_activity_maybe_truncate_entry', '__return_false' );
```

Then we need to parse the text from the activity and truncate the text-only
we can use `bp_activity_truncate_entry` hook to achieve the requirement.

**Comment by umeshnevase** [comment]
@pooja-muchandikar 
I'm working on it and could not find the way to separate the image from content, will update here once I back to this task.

**Comment by umeshnevase** [comment]
@pooja-muchandikar 
This is my current branch
https://github.com/umeshnevase/rtMedia/tree/feature/GH-1535-activity-show-image-separate-option

**Comment by SID177** [comment]
There is no way to separate image from content without DOM manipulation. So I used DOMDocument php class.
PR - https://github.com/rtMediaWP/rtMedia/pull/1544

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested with `develop` branch and the fix added looks fine but in console Ajax errors are shown. Please check those errors.

![Screenshot from 2020-02-05 09-37-51](https://user-images.githubusercontent.com/41000648/73810358-4bf68800-47fc-11ea-9552-d5d949a3bdfb.png)

---

![Screenshot from 2020-02-05 09-38-38](https://user-images.githubusercontent.com/41000648/73810360-4e58e200-47fc-11ea-9a28-9609d062f590.png)

---

cc: @thrijith

**Comment by SID177** [comment]
Its DOMDocument's `loadHTML` error, fixing with `libxml_use_internal_errors` for now.
https://stackoverflow.com/questions/6090667/php-domdocument-errors-warnings-on-html5-tags

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested with `develop` branch and the fix added looks fine with no errors/warnings. Checked with both Group activity and profile activity page.

```
Test Environment
WP: v5.3.2
rtMedia: v4.5.11 ( develop branch )
BuddyPress: v5.1.2
Theme: 2020, 2019
```

**Comment by pooja-muchandikar** [comment]
```
Test Environment
WP: v5.3.2
rtMedia: v4.5.11 ( develop branch )
BuddyPress: v5.1.2
Theme: 2020, 2019
```
Tested with **develop** branch and the fix added looks fine with no errors/warnings. Checked with both Group activity and profile activity page with BuddyPress Legacy and Nouveau template.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.0`
